barcodescanner/src/test/java/com/mindera/skeletoid/barcodescanner/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.mindera.skeletoid.barcodescanner
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
```

Comment:
Are we going to delete this ?

Reply:
They are needed to pass the travis CI tests, in the future i'll do some proper testing